P5/Exemplars/tei_rdfa.odd

```diff
@@ -0,0 +1,227 @@
+            <attDef mode="add" ident="resource">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-resource</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a SafeCURIEorCURIEorIRI for expressing
+                the partner resource of a relationship that is not intended to be navigable (e.g., a
+                'clickable' link) (also an 'object')</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="rev">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-rev</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a white space separated list of
+                TERMorCURIEorAbsIRIs, used for expressing reverse relationships between two
+                resources (also 'predicates')</desc>
+              <datatype maxOccurs="unbounded">
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="src">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-src</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">an IRI for expressing the partner
+                resource of a relationship when the resource is embedded (also a 'resource
+                object')</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="typeof">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-typeof</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a white space separated list of
+                TERMorCURIEorAbsIRIs that indicate the RDF type(s) to associate with a
+                subject</desc>
+              <datatype maxOccurs="unbounded">
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="vocab">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-vocab</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">an IRI that defines the mapping to use
+                when a TERM is referenced in an attribute value. See General Use of Terms in
+                Attributes and the section on Vocabulary Expansion.</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+          </attList>
+        </classSpec>
+
+        <!--
+          Hook the new class `att.global.analytic.rdfa` into the existing
+          class `att.global.analytic`
+        -->
+        <classSpec type="atts" ident="att.global.analytic" mode="change">
+          <classes mode="change">
+            <memberOf key="att.global.analytic.rdfa" mode="add"/>
+          </classes>
+        </classSpec>
+        
+        <elementSpec ident="citeData" mode="change">
```

Comment:
Deleting `@property` from `<citeData>` isn't a viable solution going forward. The attribute on citeData is (almost) precisely the same thing, except that it doesn't play a role in defining a proper triple (it specifies how to use the document to construct triples). Do these attributes have to be absolutely global? Can we simply override the definition in this case, as we do in various places with `@type`, for example. I feel like some thinking about the scope of RDFa in TEI might be worthwhile.